crates/sui-core/src/unit_tests/gas_tests.rs

```diff
@@ -337,7 +338,7 @@ async fn test_oog_computation_oog_storage() -> SuiResult {
 // - computation ok, OOG for storage, minimal storage ok
 #[tokio::test]
 async fn test_computation_ok_oog_storage_minimal_ok() -> SuiResult {
-    const GAS_PRICE: u64 = 30;
+    const GAS_PRICE: u64 = 1000;
```

Comment:
fwiw I wanted to use values different than 1 or 1000, but it's all good

Reply:
1001